packages/core/src/components/plekinfo-card/plekinfo-card.scss

```diff
@@ -82,7 +82,6 @@ del.dso-plekinfo-card-container {
 ::slotted([slot="meta"]) {
   display: flex !important;
   justify-content: space-between !important;
-  margin-inline-start: auto !important;
```

Comment:
Waarom zat dit je in de weg?

Reply:
Het label moet naast de titel komen. Deze drukt het element naar rechts en is dus niet meer nodig. Zie het design